modules/benchmark/R/load_data.R

```diff
@@ -58,16 +59,6 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
   
   vars_used <- format$vars[vars.used.index, ]
   
-  # check wide format and transform to long
-  if(any(duplicated(vars_used$bety_name))){
```

Comment:
Just to be clear, I'm not going to approve this PR until the wide/long transform code is restored or I get an OK from @istfer that it can be removed